microlens-pro/src/Lens/Micro/Pro/TH.hs

```diff
@@ -0,0 +1,560 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE Trustworthy #-}
+
+#if __GLASGOW_HASKELL__ >= 800
+{-# OPTIONS_GHC -Wno-trustworthy-safe #-}
+#endif
+
+#if __GLASGOW_HASKELL__ >= 800
+{-# LANGUAGE TemplateHaskellQuotes #-}
+#else
+{-# LANGUAGE TemplateHaskell #-}
+#endif
+
+#ifndef MIN_VERSION_template_haskell
+#define MIN_VERSION_template_haskell(x,y,z) 1
+#endif
```

Comment:
What is this for? I don't see it referenced anywhere

Reply:
Not sure. The entire TH module was cut-and-pasted from #107. Probably safe to remove...?

Reply:
I see it comes from [here](https://github.com/ekmett/lens/blob/master/include/lens-common.h) and is used in a [couple places](https://github.com/search?q=repo%3Aekmett%2Flens%20MIN_VERSION_template_haskell&type=code) in the lens repo (but not here). Let me try to remove it.